app/views/permit_hub_mailer/02a_notify_reviewer_application_received_email.html.erb

```diff
@@ -170,7 +170,7 @@ table[class="body"] .article {
                         
 
                         <ol style="font-family: 'Open Sans', sans-serif; font-size: 16px; line-height: 26px; font-weight: normal; margin: 0; margin-bottom: 24px; padding-left: 24px;">
-                          <li style="list-style-position: initial; margin-left: 5px;">Log into the <a href="https://buildingpermit.gov.bc.ca" style="color: #1a5a96; text-decoration: underline;">Building Permit Hub</a></li>
+                          <li style="list-style-position: initial; margin-left: 5px;">Log into the <a href="<%= @root_url %>" style="color: #1a5a96; text-decoration: underline;">Building Permit Hub</a></li>
```

Comment:
should this link be `"<%= FrontendUrlHelper.frontend_url("/login")` same as the link below or is this intended?